starky/src/get_challenges.rs

```diff
@@ -299,17 +617,24 @@ impl<const D: usize> StarkProofTarget<D> {
             Some(trace_cap)
         };
 
-        get_challenges_target::<F, C, D>(
+        get_challenges_target::<F, C, S, D>(
             builder,
+            stark,
+            public_inputs,
             challenger,
             challenges,
             trace_cap,
             auxiliary_polys_cap.as_ref(),
             quotient_polys_cap.as_ref(),
+            ctl_vars,
             openings,
             commit_phase_merkle_caps,
             final_poly,
             *pow_witness,
+            degree_bits,
+            self.degree_bits,
+            S::COLUMNS,
+            openings.auxiliary_polys.as_ref().map_or(0, |aux| aux.len()),
```

Comment:
`openings` are arguments of the method already so can be extracted internally
`S::COLUMNS` can also be inferred internally